configs/config.dell_optiplex_9010_sff

```diff
@@ -1,10 +1,27 @@
-CONFIG_USE_OPTION_TABLE=y
-CONFIG_USE_BLOBS=y
+CONFIG_LOCALVERSION="Dasharo (coreboot+SeaBIOS) v0.1.0"
+CONFIG_BOOTSPLASH_IMAGE=y
+CONFIG_BOOTSPLASH_FILE="dasharo-black.bmp"
+CONFIG_BOOTSPLASH_COMPRESS_LZMA=y
 CONFIG_VENDOR_DELL=y
 CONFIG_BOARD_DELL_OPTIPLEX_9010=y
 CONFIG_ONBOARD_VGA_IS_PRIMARY=y
-# CONFIG_DRIVERS_UART_8250IO is not set
-CONFIG_PCIEXP_CLK_PM=y
+# CONFIG_DRIVERS_INTEL_WIFI is not set
 CONFIG_SEABIOS_PS2_TIMEOUT=3000
+CONFIG_GENERIC_LINEAR_FRAMEBUFFER=y
+CONFIG_TPM_MEASURED_BOOT=y
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL_8=y
 CONFIG_POST_DEVICE_LPC=y
-CONFIG_HAVE_EM100_SUPPORT=y
+CONFIG_SEABIOS_REVISION=y
+CONFIG_SEABIOS_REVISION_ID="rel-1.16.0"
```

Comment:
@miczyg1 isn't rel-1.16 have NVME support? We need some consistent policy for what we're using where. Are you using v1.0.5 in  your setup?

Reply:
I was using v1.0.5 in the old v0.1.0 PR and it also should have NVMe support

Reply:
@miczyg1 shall I assume v1.0.5 is correct version or we should be as close to upstream as possible? Are there any reason to not use upstream? Serial console seem to work for me, I'm not sure how to test iPXE prompt issue and XHCI.

Reply:
Some of the front panel USB ports didn't work IIRC. Try using serial console without display connected and enter the boot menu with ESC, you should see nothing (no boot options), cause serial console handler is not reenabled after bootsplash is probed. iPXE prompt should not be displayed before Press ESC (...), that's the fix. Otherwise it will display the ugly string from option ROM being run.

Reply:
@miczyg1 this is not the case in my situation. Menu with iPXE option and TPM display perfectly fine.

Reply:
Okaay. I will have a look on my machine...

Reply:
@miczyg1 did you managed to check that?